Alarm the potential escape risk (#4776)
sandboxie-plus/Sandboxie: Sandboxie/msgs/Sbie-English-1033.txt

3243;txt;01
Note:  The program will continue to execute under the supervision

> of Sandboxie, even after Administrator privileges have been granted.But if SysCallLockDown is not enabled, it could lead to a program escape from the sandbox!!!

Reviewer: ```suggestion
of Sandboxie, even after Administrator privileges are granted. However,
if "SysCallLockDown" is not enabled, the program could escape from the sandbox!
```

3243;txt;01
Note:  The program will continue to execute under the supervision

> of Sandboxie, even after Administrator privileges are granted. However,
> if "SysCallLockDown" is not enabled, the program could escape from the sandbox!

Reviewer: ```suggestion
of Sandboxie even after Administrator privileges are granted. However,
granting elevated access may increase the risk of the application
exploiting an unpatched Windows vulnerability or a vulnerable driver,
potentially allowing it to escape the sandbox.
```
